src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -214,7 +214,13 @@ def get_tags(
     ):
         default_tags = set()
```

Comment:
Not something you have to fix, but just wanted to get us thinking about the lines of code above. I think we could simplify it to be easier to read and understand, and hopefully less overrides.

Reply:
after GA there will definitely be a big cleanup